config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do  
   get '/auth/:provider/callback', to: 'sessions#create'
-  delete 'logout', to: 'sessions#destroy'
+  delete 'sign_out', to: 'sessions#destroy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.